pkgs/os-specific/bsd/netbsd/default.nix

```diff
@@ -281,11 +281,12 @@ in lib.makeScopeWithSplicing
 
   # HACK: to ensure parent directories exist. This emulates GNU
   # install’s -D option. No alternative seems to exist in BSD install.
-  install = let binstall = writeText "binstall" ''
-    #!${stdenv.shell}
-    for last in $@; do true; done
+  install = let binstall = writeScript "binstall" ''
+    #!${runtimeShell}
```

Comment:
Could just change to `writeShellScript` and make the shebang automatic if you want.

Reply:
good point!